src/core/MetaTypeRegistration.cpp

```diff
@@ -55,7 +55,7 @@ struct MetaTypeRegistration
     MetaTypeRegistration()
     {
         qRegisterMetaType< size_t >( "size_t" );
-        qRegisterMetaType< Event >( "Event" );
+        qRegisterMetaType< deflect::Event >( "Event" );
```

Comment:
There is some inconsistency that we use the fully qualifying deflect::Event but only PixelStreamSegment. Not sure which one to settle for?